app/components/helpers.js

```diff
@@ -0,0 +1,33 @@
+import SCForm from './SCForm';
+import { _ } from 'lodash';
+let err_arr = [];
+
+export function nanErr(fieldValue) {
+  _.isNaN(fieldValue) ? this.state.notANumErr : null;
```

Comment:
if the value entered is not a number return the not a number error from the state.